tests/misc/CachingTest.py

```diff
@@ -146,13 +146,11 @@ def test_caching_multi_results(self):
                 retval, output = execute_coala(
                    coala.main,
                    'coala',
-                   '-c', filename + '.coafile',
-                   '-f', filename + 'test.py')
+                   '-c', filename + '.coafile')
```

Comment:
Is `coala -f filename` (without `-b`) legal?

If so, then this change here shouldnt be necessary.

If not, have you created a test to check it emits an error message?